### PR TITLE
pythonPackages.testfixtures: 6.2.0 -> 6.3.0

### PR DESCRIPTION
**Pull request by marsam**
###### Motivation for this change
fixed build

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/testfixtures/default.nix [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
   checkPhase = ''
     # django is too much hasle to setup at the moment
-    pytest --ignore=testfixtures/tests/test_django testfixtures/tests
+    pytest -W ignore::DeprecationWarning --ignore=testfixtures/tests/test_django testfixtures/tests
   '';
 
   meta = with stdenv.lib; {
-    homepage = "https://github.com/Simplistix/testfixtures";
+    homepage = https://github.com/Simplistix/testfixtures;
+    description = "TestFixtures is a collection of helpers and mock objects that are useful when writing unit tests or doc tests";
```
Comment (dotlambda) [inline pkgs/development/python-modules/testfixtures/default.nix]: https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

**Review by worldofpeace — COMMENTED** [review]
I've built all proper dependents locally with success using `nix-review`.
The only caveat  was `python37Packages.scrapy` which doesn't support python37.

File: pkgs/development/python-modules/testfixtures/default.nix [hunk 2]
```diff
@@ -1,23 +1,32 @@
...
   src = fetchPypi {
     inherit pname version;
-    sha256 = "7e4df89a8bf8b8905464160f08aff131a36f0b33654fe4f9e4387afe546eae25";
+    sha256 = "1x16xkw483nb1ngv74s7lgaj514pb1ldklal7kb7iwqbxcgnrh2k";
   };
 
-  checkInputs = [ mock manuel pytest sybil zope_component ];
+  checkInputs = [ pytest mock sybil zope_component ];
+
+  patches = [
```
Comment (worldofpeace) [inline pkgs/development/python-modules/testfixtures/default.nix]: Please add a comment with information like:

* why it's needed
* if/when it should be dropped

Comment (Mic92) [reply pkgs/development/python-modules/testfixtures/default.nix]: i.e.: fix tests for python3.7 & can removed for the next release.

File: pkgs/development/python-modules/testfixtures/default.nix (hunk 1 above)
Comment (marsam) [reply pkgs/development/python-modules/testfixtures/default.nix]: Fixed

**Comment by dotlambda** [comment]
Iincluded in https://github.com/NixOS/nixpkgs/pull/49741. Thanks a lot!

**Pull request closed without merging**